<launch>
 <node pkg="autonomous_mobile_robot_2022" name="autonomous_mobile_robot_2022_pm" type="autonomous_mobile_robot_2022_pm" output="screen">
  <param name="ROBOT_NAME" value="megarover"/>
  <!--megarover turtlebot3-->
  <param name="PATH_PLANNING_METHOD" value="potential_method"/>
  <!--potential_method csv-->
  <param name="PATH_PLANNING_FILE" value="$(find autonomous_mobile_robot_2022)/path/-sin.csv"/>
  <param name="USE_AMCL" value="true"/>
  <param name="IS_SIMULATOR" value="true"/>
  <param name="PUBLISH_COMMAND" value="true"/>
  <param name="PID_CONTROL" value="false"/>
  <param name="ANGLE_CORRECTION" value="true"/>
  <param name="MAX_VELOCITY" value="0.5"/>
  <param name="MAX_ANGULAR" value="3.141592653589793"/>
  <param name="TARGET_POSITION_X" value="-9.5"/><!--DESTINATION  -9.5, -8.0-->
  <param name="TARGET_POSITION_Y" value="-8.0"/>
  <param name="GAIN_PROPORTIONAL" value="1.0"/>
  <param name="GAIN_INTEGRAL" value="10.0"/>
  <param name="GAIN_DIFFERENTIAL" value="0.001"/>
  <param name="AVOIDANCE_RADIUS" value="0.3"/>
  <param name="PATH_TRACKING_MARGIN" value="0.2"/>
  <param name="PATH_CREATE_PERIOD" value="1"/>
  <param name="POTENTIAL_FIELD_WIDTH" value="2"/>
  <param name="POTENTIAL_FIELD_DIVDE_X" value="20"/>
  <param name="POTENTIAL_FIELD_DIVDE_Y" value="20"/>
  <param name="CALCULATE_BIAS" value="true"/>
  <param name="POTENTIAL_BIAS_COEFFICIENT_0" value="-1.0"/>
  <param name="POTENTIAL_BIAS_COEFFICIENT_1" value="0.5"/>
 </node>
</launch>
